common/src/main/java/org/conscrypt/OpenSSLExtendedSessionImpl.java

```diff
@@ -70,17 +66,6 @@ AbstractOpenSSLSession getDelegate() {
         };
     }
 
-    /* @Override */
-    @SuppressWarnings("MissingOverride") // For Android backward-compatibility.
-    public List<SNIServerName> getRequestedServerNames() {
```

Comment:
This needs to be here to return something. Otherwise Java 8 clients will get NoSuchMethodException. It could return List<Object> and pass during runtime.